portal/src/main/resources/application.conf

```diff
@@ -42,7 +42,9 @@ database {
     path = ${?CMWL_GITLAB_USER_NAME}
     token = "token"
     token = ${?CMWL_GITLAB_TOKEN}
-    defaultFileVersion = "v0.0.1"
+    defaultFileVersion = "v.0.0.1"
     defaultFileVersion = ${?CMWL_DEFAULT_FILE_VERSION}
+    defaultBranch = "master"
+    defaultBranch = ${?CMWL_DAFAUTL_BRANCH}
```

Comment:
Hey!  You got a typo.
You wrote DAFAUTL instead of DEFAULT

Reply:
Thx, will fix it asap